### ISSUE TITLE
there should be coordination in output on x-term console and output of beryl console. 

### ISSUE THREAD

**Issue by shirishag75** [mars-sim/mars-sim#149]
x-term console says one thing, beryl console says another - 

On x-term console I get these messages - 

`2018-11-22T06:34:13.4 (Severe) [x8] PhysicalCondition : [Fire Star] Larry Ponda has insufficient oxygen.`

Apparently Larry Ponda is on Alpha Base . I don't know what Fire Star means . 

It auto-completes on MarsNet but says - 

Connected with MarsNet > ​Fire Star

System : I do not recognize any person or settlement by the name of 'Fire Star'.

On beryl console it says - 

Connected with MarsNet > ​Larry Ponda

Connected with Larry Ponda > ​association

You : What is your associated settlement ?
Larry Ponda: My associated settlement is Alpha Base

Connected with Larry Ponda > ​doing 

You : What are you doing ?
Larry Ponda: Relaxing

Now what

**Comment by mokun** [comment]
> Update - I had a look before updating to the new version and having a new simulation run. There was no vehicle or person by name of Fire Star on any of the bases :(

Do you mean that the Fire Star rover initially exists in a simulation. However, you saved the simulation and quit. Then you restart mars-sim and load from a saved game. At this point, the Fire Star rover no longer exists ?

**Comment by shirishag75** [comment]
I don't know. I tried to look into all the settlements during the simulation itself and didn't see a Fire Star rover in any settlement. I looked into all the three settlements, it seems/seemed like a ghost rover.

**Comment by mokun** [comment]
You can type in `vehicle` when connecting to a settlement to see a list of vehicles

For now, this is what we have : 

```
Connected with MarsNet > ​Alpha Base

System : You are now connected with Alpha Base. What would like to know ?

Connected with Alpha Base > ​vehicle

You : What are the vehicles in the settlement ? 

   < Name >          < Type >            < Mission >           < Lead >
  ------------------------------------------------------------------------
  Odyssey           Cargo Rover         Regolith Prospecting  James Silkworth
  Oasis             Explorer Rover                             
  Areos Aster       Explorer Rover                             
  LUV 001           LUV                                        
  LUV 002           LUV                                        
  LUV 003           LUV                                        
  Windstorm         Transport Rover                            

 ----------------------------------------------------
                             Total # of Rovers : 7
 ----------------------------------------------------
                  # of Cargo Rovers on Mission : 0
              # of Transport Rovers on Mission : 0
               # of Explorer Rovers on Mission : 0
 # of Light Utility Vehicles (LUVs) on Mission : 0
 ----------------------------------------------------
                        # of Rovers on Mission : 0
 ----------------------------------------------------
              # of Parked/Garaged Cargo Rovers : 1
          # of Parked/Garaged Transport Rovers : 1
           # of Parked/Garaged Explorer Rovers : 2
                      # of Parked/Garaged LUVs : 3
 ----------------------------------------------------
                    # of Rovers NOT on mission : 7
```

**Comment by shirishag75** [comment]
I'm back. The point is/was that every vehicle needed to be associated to be with a settlement. I had looked into every settlement and Fire Starter was not found in any of the three settlements. It was not listed. 

There should be some logic that binds each vehicle with a settlement or two (if need be.)

**Comment by mokun** [comment]
> There should be some logic that binds each vehicle with a settlement or two (if need be.)

Yes agree. Players should be able to gain insight into which `associated settlement` to which the rover is registered and which `settlement` it's currently parked or garage at. 

In the next few builds, we'll add new keywords for knowing the current status of rover, its specs and its historical missions, etc. 

Do provide feedback and comments along the way on how well the implementation works for you.

**Comment by mokun** [comment]
Currently, at MarsNet, type "vehicle" and you will see something as follows : 

```
Connected with MarsNet > ​vehicle

System : Here's the roster list of vehicles in each settlement.
 ----------------------------------------- 
                Alpha Base
 ----------------------------------------- 
Windstorm                Cargo Rover
Dust Devil               Cargo Rover
Fire Star                Explorer Rover
Shooting Star            Explorer Rover
Merih                    Explorer Rover
LUV 001                  Light Utility Vehicle
LUV 002                  Light Utility Vehicle
LUV 003                  Light Utility Vehicle
Oasis                    Transport Rover
Osprey                   Transport Rover

 ----------------------------------------- 
               Plymouth Base
 ----------------------------------------- 
Desert Fox               Explorer Rover
Blodstjarna              Explorer Rover
LUV 004                  Light Utility Vehicle

 ----------------------------------------- 
            Schiaparelli Point
 ----------------------------------------- 
Spirit II                Explorer Rover
Bahram                   Explorer Rover
LUV 005                  Light Utility Vehicle

```

**Comment by shirishag75** [comment]
I saw that . I did see that. It would be interesting to see the specs of the vehicles and have a report about them, the state of them etc. so we have real-time info. on the state of the vehicles themselves.

**Comment by mokun** [comment]
See the new [Vehicles](https://github.com/mars-sim/mars-sim/wiki/Vehicles) wiki

**Comment by mokun** [comment]
I found an error is calculating the range of the vehicle. 

For a typical BEV, the battery capacity is in kWh.

In our case, we use `Solid Oxide Fuel Cell (SOFC)` because `methane` is readily available from `Sabatier Reactor`.

Before, I totally ignore the `Specific Energy` of methane, which is 15.416kWh / kg or 55.5 MJ / kg.
> Note : 3.6 MJ = 1 kWh

When I factored that in, the range of each type of vehicle is now a few times farther, which is what it used to be in v3.07.

> Note : the total energy available from a full tank of methane =  METHANE_SPECIFIC_ENERGY * fuelCapacity * SOFC_CONVERSION_EFFICIENCY * drivetrainEfficiency / 100.0;

**Comment by bevans2000** [comment]
> If it's too verbose, we can find ways to improve on how we present them.

Change the bottom 10 values into table having 2 rows and 6 columns. Each row uses the same units
